libs/domain-service/src/resolvers/index.ts

```diff
@@ -8,6 +8,23 @@ import {
   getRegistriesForDomain,
 } from "../registries";
 
+Promise.allSettled =
```

Comment:
Why is it needed ?

Reply:
allSettled doesn't exist in react-native env but only on node